src/app/modules/admin/components/orders/orders.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="orders-wrap">
+    <div class="orders-wrap__search">
+        <div class="orders-wrap__search-number">
+            <label class="search-number-label" for="search-number"
+                >Order number:</label
+            >
+            <input class="search-number-input" type="text" id="search-number" />
+        </div>
+        <div class="orders-wrap__search-word">
+            <label class="search-word-label" for="search-word"
+                >Product name:</label
+            >
+            <input class="search-word-input" type="text" id="search-word" />
+        </div>
+    </div>
+    <div class="orders-items-wrap">
+        <div class="orders-wrap__products">
+            <div class="orders-wrap__products-header">
+                <div class="orders-wrap__products-header-order">
+                    <span>Order №: <b>303030</b></span>
+                    <span>Ordering time №: <b>7 february 2020 14:20</b></span>
+                </div>
+                <div class="orders-wrap__products-header-shop">
+                    <span class="shop-label">Shop: <b>Shop Name</b></span>
+                    <div class="links-wrap">
+                        <a href="#">Go to the shop</a>
+                        <a href="#">
+                            <mat-icon>
+                                <span> email </span>
+                            </mat-icon>
+                            Message to the seller
+                        </a>
+                    </div>
+                </div>
+                <div class="orders-wrap__products-header-total">
+                    <span class="order-title">Total:</span>
+                    <span class="order-total"><b>$7.77</b></span>
+                </div>
+                <div class="orders-wrap__products-header-remove">
+                    <mat-icon>delete</mat-icon>
+                </div>
+            </div>
+            <div class="orders-wrap__products-list">
+                <ul class="orders-wrap__products-list-items">
+                    <li class="item">
+                        <img
+                            src="../../../../../assets/img/avatar.png"
```

Comment:
i believe we should get this data from API...
so u need to create orders.state, orders.reducers, orders.actions, orders.selectors